common/src/main/java/io/netty/util/concurrent/AbstractEventExecutor.java

```diff
@@ -177,25 +177,19 @@ protected static void runTask(@Execute Runnable task) {
     /**
      * Like {@link #execute(Runnable)} but does not guarantee the task will be run until either
      * a non-lazy task is executed or the executor is shut down.
-     *
-     * This is equivalent to submitting a {@link AbstractEventExecutor.LazyRunnable} to
-     * {@link #execute(Runnable)} but for an arbitrary {@link Runnable}.
-     *
+     * <p>
      * The default implementation just delegates to {@link #execute(Runnable)}.
+     * </p>
      */
     @UnstableApi
     public void lazyExecute(Runnable task) {
```

Comment:
should this be execute(task, false)?

Reply:
check `SingleThreadEventExecutor::lazyExecute` overridden method:

```java
    @Override
    public void lazyExecute(Runnable task) {
        lazyExecute0(task);
    }

    private void lazyExecute0(@Schedule Runnable task) {
        execute(ObjectUtil.checkNotNull(task, "task"), false);
    }
    ```